### DISCUSSION TITLE
Angular clr-icon state is unclear

### DISCUSSION THREAD

**Discussion by ph55** [vmware/clarity#5817] [Ideas] [answered]
There is a warning in official [docs](https://clarity.design/foundation/icons/)
![image](https://user-images.githubusercontent.com/1386622/114056094-fef7c100-9899-11eb-98bc-bf9e1577cc0b.png)

But there is still many references to "old" angular format:

https://clarity.design/angular-components/vertical-nav/#code-examples
https://clarity.design/angular-components/alert/#types-2
https://clarity.design/angular-components/wizard/

And I'm sure there is more places.

So to sum it up - 
It's unclear if it's deprecated and not recommended to use or still valid option.

If it's still valid option, it should be added to angular docs (not old one) IMO.
If it's deprecated, it should be removed from angular component examples and replaced with new one.

**Comment by hippee-lee** [comment, marked as answer]
Hi @ph55 -,

`clr-icon` is depricated in angular components as of v5. There were a number of changes leading up to the v5 release related to icons. 

1. An [eslint](https://github.com/vmware/clarity/pull/5331) rule was added to facilitate migrating: 
2. 2. `@clr/angular@5.x.x` [replaced](https://github.com/vmware/clarity/pull/5190) internal uses of `clr-icon` with `cds-icon` 
3. Deprecated icon styles were [removed](https://github.com/vmware/clarity/pull/5110) from cds-icons making it a clean break from v4 -> v5

With that said, you can still use `clr-icon` with `@clr/angular@5.x.x `components as described in the angular components documentation. If you find a scenario that isn't working the way it did in v4, before the deprication, please open an issue with a small reporduction so we can triage and address it.

**Reply by ph55** [reply]
I see. Thanks.
Also found this article that explains icon progress -
https://medium.com/claritydesignsystem/goodbye-clarity-icons-hello-clarity-core-adb07de9739e

Don't you think though that `clr-icon` should be replaced with `cds-icon` in angular components examples to avoid confusion ?

**Reply by hippee-lee** [reply]
Great point and you are right. I'll see if I can get a PR for that today.

**Reply by hippee-lee** [reply]
Thanks for the feedback. Here is a PR in case you want to track it: https://github.com/vmware/clarity/pull/5820
